analyzers/tests/SonarAnalyzer.UnitTest/TestCases/MultilineBlocksWithoutBrace.old.cs

```diff
@@ -0,0 +1,173 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Tests.TestCases
+{
```

Comment:
You can remove this namespace from the old UTs

Reply:
See above, I wan't to drop this file. The cases are unclear, and are all covered by the new file.